data/helpers.d/utils

```diff
@@ -24,3 +24,19 @@ ynh_get_plain_key() {
         fi
     done
 }
+
+# Curl abstraction to simplify its call
+#
+# Helpfully mostly for curl arguments
+#
+# example: ynh_curl_abstract "value1=$var1&value2=$var2" "/install.php?installButton"
+# 
+# usage: ynh_curl_abstract "data_post" "url_access"
+# | arg: data_post - POST request to send
```

Comment:
I get your point. I was suggesting that one is not passing POST raw data as such, but a string that represents the data to fit into the variables in the POST request. Not a big deal :-)